conans/model/settings.py

```diff
@@ -298,3 +298,25 @@ def dumps(self):
         """
         return "\n".join(["%s=%s" % (field, value)
                           for (field, value) in self.values_list])
+
+    def get_definition_values(self, name):
```

Comment:
Why you don't love this? The name of the method? the implementation?

Reply:
I think both hahah

Reply:
Probably because we only want the internal definition?
I mean, instead of ``settings.get_definition_values("compiler.gcc.cppstd")``, which is the current approach, something more like: ``settings.get_definitions()`` return the barebones ``dict``, so: 
```
definitions = settings.get_definitions()
cppstd = definitions["gcc"]["cppstd"]  # maybe protect with ``try-except KeyError``
```
Maybe we also want to have a direct access to the ``settings_yaml`` definition, and not via de ``conanfile``? Just wondering, probably not.

Reply:
> settings.get_definitions()

Yes, that would definitely be better.